CONTRIBUTING.md

```diff
@@ -116,7 +116,8 @@ Implementing Features
 --------
 
 It's likely that you'll have to implement feature requests or enhancements on your own. To do that, you'll need
-to be comfortable with JavaScript, Promises, Node.js, and familiar with [Esprima's abstract syntax tree format](http://esprima.org/demo/parse.html#).
+to be comfortable with JavaScript, Promises, Node.js, and familiar with Javascript abstract syntax trees (specifically Esprima and Bablyon).
+Check out [AST Explorer](http://astexplorer.net/).
```

Comment:
Yeah it's pretty cool - you can live edit babel plugins. Would be cool to do the same for jscs rules https://github.com/fkling/esprima_ast_explorer/issues/26. Just added support for a lot more parsers too.